static/src/stylesheets/head.content.scss

```diff
@@ -24,6 +24,7 @@
 @import 'module/_chapters';
 @import 'module/content/_article-immersive';
 @import 'module/content/_article-minute';
+@import 'module/content/_article-labour-liverpool';
```

Comment:
yeah that's right. maybe a public method that exposes the inline method? we don't really want to add a new variable for each new design i don't think? @johnduffell might have some good advice on this